src/Client.java

```diff
@@ -21,5 +22,141 @@ public static void main(String args[]) throws Exception{
+    }
+
+    public void processOpt1(CenterServer stub) throws RemoteException {
+        Scanner scanner = new Scanner(System.in);
+        TeacherRecord teacher = new TeacherRecord();
+        System.out.println("Please input teacher's first name:");
+        teacher.setFirstName(scanner.nextLine().trim());
+        System.out.println("Please input teacher's last name:");
+        teacher.setLastName(scanner.nextLine().trim());
+        System.out.println("Please input teacher's address");
+        teacher.setAddress(scanner.nextLine().trim());
+        System.out.println("Please input your teacher's specialization:");
+        teacher.addSpecialiazation(scanner.nextLine().trim());
+        System.out.println("Please input teacher's location:");
+        teacher.setLocation(scanner.nextLine().trim());
+        System.out.println("Please input teacher's phone:");
+        teacher.setPhone(scanner.nextInt());
+        stub.createTRecord(teacher.getFirstName(), teacher.getLastName(), teacher.getAddress(), teacher.getPhone(), teacher.getSpecialiazation(), teacher.getLocation());
+    }
+
+    public void processOpt2(CenterServer stub) throws RemoteException {
+        Scanner scanner = new Scanner(System.in);
+        StudentRecord student = new StudentRecord();
+        System.out.println("Please input student's first name:");
+        student.setFirstName(scanner.nextLine().trim());
+        System.out.println("Please input student's last name:");
+        student.setLastName(scanner.nextLine().trim());
+        System.out.println("Please input student's status:");
+        student.setStatus(scanner.nextLine().trim());
+        System.out.println("Please input your student's statusDate:");
+        student.setStatusDate(scanner.nextLine().trim());
+        System.out.println("Please input student's courses(split with space):");
+        for (String s :
+                scanner.nextLine().split(" ")) {
+            student.getCoursesRegistered().add(s);
+        }
+        stub.createSRecord(student.getFirstName(), student.getLastName(), student.getCoursesRegistered(), student.getStatus(), student.getStatusDate());
+    }
+
+    public int processOpt3(CenterServer stub) {
+        //TODO: We should let this function return int and remove the argument "managerId".
+//        stub.getRecordCounts();
+        return 0;
+    }
+
+    public void processOpt4(CenterServer stub) throws Exception {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Please input your record id:");
+        String recordId = scanner.nextLine().trim();
+        System.out.println("Please input the field name you want to change:");
+        String fieldName = scanner.nextLine().trim();
+        System.out.println("Please input new value:");
+        String newValue = scanner.nextLine().trim();
+        stub.editRecord(recordId, fieldName, newValue);
+    }
+
+
+    public void verifyId(String managerId) throws Exception {
```

Comment:
what is it for? we should address this validation with case in the beginning right?